onnxruntime/core/framework/execution_frame.h

```diff
@@ -50,8 +50,9 @@ class IExecutionFrame {
   OrtValue* GetMutableNodeInputOrOutputMLValue(int index);
 
 #ifdef ENABLE_TRAINING
-  // Override the index-th output with ort_value
-  Status SetOutputMLValue(int index, const OrtValue& ort_value);
+  void UpdateFeeds(const std::vector<int>& feed_mlvalue_idxs, const std::vector<OrtValue>& feeds);
```

Comment:
Can this function fail? Does it need to return Status?

Reply:
It can fail if asserts fail, no need for status.